src/poggit/module/HtmlModule.php

```diff
@@ -41,7 +41,7 @@ protected function headIncludes(string $title, $description = "", $type = "websi
       <meta name="keywords"
             content="<?= implode(",", array_merge([Mbd::esq($title)], $extraKeywords)) ?>,plugin,PocketMine,pocketmine plugins,MCPE plugins,Poggit,PocketMine-MP,PMMP"/>
       <meta property="og:site_name" content="Poggit"/>
-      <meta property="og:image" content="<?= $image ?? (Meta::getSecret("meta.extPath") . (date("M j") === "Apr 1" ? "res/mrs-poggit.png" : "res/poggit-icon.png")) ?>"/>
+      <meta property="og:image" content="<?= $image ?? (Meta::getSecret("meta.extPath") . (date("M j") === "Apr 1" ? "res/mrs-poggit.png" : "res/poggit-icon.gif")) ?>"/>
```

Comment:
See L56 too.

Reply:
> See L56 too.

Already covered by https://github.com/HBIDamian/poggit/blob/beta/res/poggit-icon.ico :P Step ahead of you, bruv! :D

Reply:
My bad, too hot to think here and saw the red decrease in file size as a deleted file 🤦 (hence the doubled legacy size confusion, sorry).